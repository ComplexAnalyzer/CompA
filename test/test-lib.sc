def main()
{ 
    a = [1,2,3];
    b = [7,3,10];
    c = [1.1, 2.2, 3.3];
    d = [4.4, 1.1, 6.6];
    f = a * b;
    (0<=i<#f) print(f[i]);
    f = a + b;
    (0<=i<#f) print(f[i]);
    f = a & b;
    (0<=i<#f) print(f[i]);
    e = c * d;
    (0<=i<#e) print(e[i]);
    e = c + d;
    (0<=i<#e) print(e[i]);
    e = c & d; 
    (0<=i<#e) print(e[i]);

}
