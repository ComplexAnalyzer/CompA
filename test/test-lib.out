3
1
2
3
7
3
10
1
2
3
7
10
1.100000
1.100000
2.200000
3.300000
4.400000
1.100000
6.600000
1.100000
2.200000
3.300000
4.400000
6.600000
